### PR TITLE
Bugfix/School 407/make-up-the-where-to-start-page

### PR DESCRIPTION
**Pull request by Tohenzoo**

### CODE REVIEW

File: components/WhereToStartPageComponents/Tittle.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Layout from "../../components/layouts/layout";
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/Tittle.tsx:1]: unused import

File: components/WhereToStartPageComponents/Tittle.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import styles from "../../components/WhereToStartPageComponents/styles/wtsp.module.css"
+import { Switch, Typography } from "antd";
+
+const { Title } = Typography;
+
+function Tittle() {
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/Tittle.tsx:8]: It's typo in word `Title` (in file name too)

File: components/WhereToStartPageComponents/WhereToStart.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import type { StepsProps } from 'antd';
+import { Steps, List, Avatar } from 'antd';
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/WhereToStart.tsx:3]: unused imports

File: components/WhereToStartPageComponents/WhereToStart.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    IdcardOutlined,
+    ScheduleOutlined,
+    UsergroupAddOutlined,
+    HomeOutlined
+} from "@ant-design/icons";
+import styles from "../../components/WhereToStartPageComponents/styles/wtsp.module.css"
+
+const data = [
+  {
+    title: <div><div className={styles.listMain}>Создание организации</div><div className={styles.listChildren}>Добавляйте свои организации, чтобы получить доступ ко всем функциям. </div></div>,
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/WhereToStart.tsx:14]: move inline css in scss file

File: components/WhereToStartPageComponents/index.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import React, { useEffect } from "react";
+import styles from "../../components/WhereToStartPageComponents/styles/wtsp.module.css";
+import { Switch, Typography } from "antd";
+import Cookies from "universal-cookie";
+import Tittle from "./Tittle";
+import WhereToStart from "../WhereToStartPageComponents/WhereToStart";
+import SelectOrganization from "../MicsComponents/SelectOrganizationComponent";
+import { useState } from "react";
+
+const { Title } = Typography;
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/index.tsx:11]: unused const

File: components/WhereToStartPageComponents/index.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import Layout from "../../components/layouts/layout";
+import React, { useEffect } from "react";
+import styles from "../../components/WhereToStartPageComponents/styles/wtsp.module.css";
+import { Switch, Typography } from "antd";
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/index.tsx:4]: unused imports

File: components/WhereToStartPageComponents/styles/wtsp.module.css [hunk 7]
```diff
@@ -0,0 +1,51 @@
+.title {
```
Comment (gaskeo) [inline components/WhereToStartPageComponents/styles/wtsp.module.css:1]: we use scss preprocessor

File: package.json [hunk 8]
```diff
@@ -37,5 +37,6 @@
     "eslint": "8.8.0",
     "eslint-config-next": "12.0.10",
     "typescript": "4.5.5"
-  }
+  },
+  "version": "1.22.20"
```
Comment (gaskeo) [inline package.json:41]: why it's 1.22.20 version number?

File: pages/whereToStartPage/index copy.tsx [hunk 9]
```diff
@@ -0,0 +1,54 @@
+import Layout from "../../components/layouts/layout";
```
Comment (gaskeo) [inline pages/whereToStartPage/index copy.tsx:1]: this file unused

**Review by gaskeo — CHANGES_REQUESTED** [review]

**Review by dowlsss — APPROVED** [review]

**Review by gaskeo — APPROVED** [review]

**Pull request merged**